trafficcloud/api.py

```diff
@@ -4,19 +4,21 @@
 import os
 import shutil
 from app_config import get_project_path, get_project_video_path, update_config_without_sections, get_config_without_sections
-import pm
 import subprocess
 import video, feat_config
 
+from pm import create_project
+
 from plotting.make_object_trajectories import main as db_make_objtraj
 
-def saveFiles(diction, *args):
-    return pm.ProjectWizard(diction).identifier
+def uploadVideo(identifier, video_filename):
```

Comment:
Note that this will break the current workflow until Deniz's changes get merged.